api/src/lib/redis/main.go

```diff
@@ -0,0 +1,17 @@
+package redis
+
+import "github.com/go-redis/redis"
+
+var client *redis.Client
+func GetClient() (*redis.Client){
+	if client != nil {
+		return client
+	}
+
+	client = redis.NewClient(&redis.Options{
```

Comment:
singleton ?

Reply:
No configuration file (multienv support)?

Reply:
@fiverrben in a real world app there we should support multi env, not here